command/agent/http.go

```diff
@@ -87,6 +88,7 @@ func NewHTTPServers(agent *Agent, config *Config, logOutput io.Writer) ([]*HTTPS
 			logger:   log.New(logOutput, "", log.LstdFlags),
 			uiDir:    config.UiDir,
 			addr:     httpAddr.String(),
+			prefix:   "https",
```

Comment:
Hey @radekg, yeah, I remember thinking that maybe using a static string like this wasn't the most optimal method, but for some reason it seemed like the least invasive. I'll review and update in a bit!